apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -464,6 +467,17 @@ type BorrowWithinCohort struct {
 	MaxPriorityThreshold *int32 `json:"maxPriorityThreshold,omitempty"`
 }
 
+// FairSharing contains the properties of the ClusterQueue when participating in fair sharing.
+type FairSharing struct {
+	// weight gives a comparative advantage to this ClusterQueue when competing for unused
+	// resources in the cohort against other ClusterQueues.
+	// The share of a ClusterQueue is based on the dominant resource usage above nominal
+	// quotas for each resource, divided by the weight.
+	// Admission prioritizes scheduling workloads from ClusterQueues with the lowest share
+	// and preempting workloads from the ClusterQueues with the highest share.
```

Comment:
As per our design, it seems that we need to default the `weight` to `1` if it is empty.
So, how to default this?

https://github.com/kubernetes-sigs/kueue/tree/main/keps/1714-fair-sharing#share-value-function-and-weights

Reply:
added the default